### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by Nitschi** [JPhilipp/powerdalle#5] [General]
Hello and thank you for your great project. It's simple, it works and we would love to continue using it!
Have you thought about adding a License to your repo?
With the MIT, Apache2 or GPL-3 License we would be certain, that we are actually allowed to use it and possibly adapt it. It doesn't really matter which one you choose.

Thank you

Alex

**Comment by JPhilipp** [comment]
Great idea, thanks! An MIT License has now been added to the project.
